### PR TITLE
Clowder doc rewrites

### PR DESCRIPTION
**Pull request by max-zilla**

### CODE REVIEW

File: clowder.md [hunk 1]
```diff
@@ -1,46 +1,102 @@
...
+1. Create a collection to hold relevant datasets (optional)
+
+    **POST /api/collections** _provide a name; returns collection ID_  
+    
+2. Create a dataset to hold relevant files and add it to the collection
+
+    **POST /api/datasets/createempty** _provide a name; returns dataset ID_  
+    **POST /api/collections/_collection id_/datasets/_dataset id_**  
+    
+3. Upload files and metadata to dataset
```
Comment (dlebauer) [inline clowder.md]: perhaps we need a separate section with examples for specific sources and sensors? e.g. what is the call that @nfahlgren will use? What needs to be done to adapt this for the field system?

**Comment by dlebauer** [comment]
@max-zilla could you summarize the Danforth pipeline and document it (e.g. and close terraref/computing-pipeline#36

File: clowder.md (hunk 1 above)
Comment (max-zilla) [inline clowder.md]: Yes, we'll likely want a separate section dedicated to specific examples - in this portion, I'm just speaking broadly about the generic approach.

**Comment by max-zilla** [comment]
I'll create a Danforth document that summarizes the pipeline and the scripts involved (will update as Noah and I refine) to address both questions.

**Comment by dlebauer** [comment]
at the end of the chapter ... like in text book "Exercises" or "Examples"

**Comment by dlebauer** [comment]
@max-zilla sorry I just opened #3 with a [few edits](https://github.com/terraref/documentation/pull/3/files) to the master branch before looking at this. Do you have time to merge these?

**Comment by max-zilla** [comment]
@dlebauer I can merge them this afternoon no problem, but I need write access to terraref/documentation to do so. Don't need write access for the other repos for now.

**Comment by dlebauer** [comment]
@max-zilla sorry I didn't realize that the developer group lacked write access to this repository. I've fixed that. 

But if you or any @terraref/developers want to use the web-based gitbook editor I can add you there. Alternatively, There is a [nice gitbook desktop editor of osx, linux, and windows](https://www.gitbook.com/editor)
